Lombiq.NodeJs.Extensions/package.json

```diff
@@ -20,7 +20,7 @@
     "postcss": "^8.4.18",
     "postcss-cli": "^9.1.0",
     "rimraf": "^3.0.2",
-    "sass": "^1.55.0",
+    "sass": "1.59.3",
```

Comment:
Shouldn't this be `"^1.59.3"` to allow updates within the same minor and patch versions?

Reply:
That allows minor version updates too, i.e. 1.60.x too. That has bitten us here and that's specifically what I want to avoid. (I'd rather have everything as a fixed version TBH, surprise updates are not good, as we've just seen.)

Reply:
I see, thank you!